mvrx-mocking/build.gradle

```
@@ -7,6 +8,14 @@ apply plugin: "jacoco"
 apply plugin: "com.vanniktech.maven.publish"
 apply plugin: 'kotlin-android-extensions'
 
+tasks.withType(KotlinCompile).all {
```

nit: for configuration avoidance:
```suggestion
tasks.withType(KotlinCompile).configureEach {
```
same for other `.all` usages